components/compositing/compositor.rs

```diff
@@ -495,6 +495,8 @@ impl<Window: WindowMethods> IOCompositor<Window> {
             }
 
             (Msg::IsReadyToSaveImageReply(is_ready), ShutdownState::NotShuttingDown) => {
+                // TODO: We cannot change this to `assert_eq` because `ReadyState` doesn't
+                //       derive `Debug`
                 assert!(self.ready_to_save_state == ReadyState::WaitingForConstellationReply);
```

Comment:
I think it should derive `Debug`, mind changing it or filing an issue for that?

Reply:
Let me derive it in this PR! Thanks!